test/alphabet/nucleotide/CMakeLists.txt

```diff
@@ -1,15 +1,15 @@
 add_executable(dna5_test dna5_test.cpp)
```

Comment:
These are all valid points, however we are working on so many things right now, that you rarely get to cover everything. See for example #44: it does not yet apply the proposed changes to the aminoacid submodule and not to the quality submodule, but it's already really huge and difficult to review.
It would be much more practical to do it in smaller units.

In our case here you including the changes is more consistent, that's true, but it's also only cosmetical in the sense that they are not required for the other changes to make sense. In my PR the changes to the file are not by design, but because they are necessary (some file were removed...), so I cannot "skip" this file.
Of course I can still resolve merge conflicts, but it's additional pain for complicated PR and seeing that both PRs will probably be merged on monday within half an hour of each other it would be *nice* if you could just skip this [you have a conflict with current master anyways].